ClavaLaraApi/src-lara/clava/clava/analysis/CheckBasedAnalyser.js

```diff
@@ -29,9 +29,10 @@ export default class CheckBasedAnalyser extends Analyser {
      */
     analyse($startNode = Query.root()) {
         const checkResultList = [];
-        for (const $node of Query.searchFrom($startNode)) {
+        for (const a of Query.searchFrom($startNode)) {
+            const $jp = a;
```

Comment:
Unnecessary?

Reply:
In the original Typescript file, the code appears as:
```typescript
const $jp = a as Joinpoint;
```
When transpiled, the type information is removed.